src/newlines.cpp

```diff
@@ -397,6 +396,7 @@ chunk_t *newline_add_between(chunk_t *start, chunk_t *end)
    }
 
    /* Scan for a line break */
+   chunk_t *pc;
```

Comment:
You are right. In this case it is actually more appropriate to use a new variable. Both variables have the same name but actually they are two different things so it is more sensible to use separate variables.
There are a few more locations where this could be done as well.